package.json

```
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/cli",
   "description": "Small package containing useful typescript utilities.",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

```suggestion
  "version": "4.1.0",
```

(normally do minor for new feature)